Marlin/src/pins/stm32f1/pins_MKS_ROBIN.h

```diff
@@ -165,6 +165,8 @@
 #define SPI_DEVICE                             2
 
 #define SDIO_SUPPORT
+#define SDIO_CLOCK                       4500000
+#define SDIO_READ_RETRIES                     16
```

Comment:
Why is this set in both the pins file and also `build_flags`?

Reply:
Is it used by framework libraries, that cannot see defines from the Marlin pins file?

Reply:
> Why is this set in both the pins file and also `build_flags`?

I migrate the build_flags from TRANSFER_CLOCK_DIV to SDIO_CLOCK. Them I added in the Matt board the value that should work for him (the same value of other mks boards)

Reply:
> Is it used by framework libraries, that cannot see defines from the Marlin pins file?

It's used only by marlin.

Reply:
I think we should be consistent and either always set it in pins files or always set in platformio.ini.

If it is only used by Marlin the pins file is the better place for it.